doc/api/buffer.md

```diff
@@ -631,7 +631,7 @@ console.log(`${str}: ${str.length} characters, ` +
 When `string` is a `Buffer`/[`DataView`]/[`TypedArray`]/[`ArrayBuffer`], the
 actual byte length is returned.
 
-Otherwise, converts to `String` and returns the byte length of string.
+Otherwise if `typeof string !== 'string'`, an exception is thrown.
```

Comment:
I agree that we can improve this, without quoting actual code. Either that, or we can simply remove this as @silverwind suggested.